src/scripts/farming/Farming.ts

```diff
@@ -277,7 +325,10 @@ class Farming implements Feature {
             50,
             3,
             [0, 0, 15, 0, 0],
+            25,
             BerryColor.Pink,
+            12.6,
```

Comment:
Mago is 4.7" in your link, displaying in game as 5.0

Reply:
Same resolution as Bluk, though I can only find a screenshot of its gen 3 tag (which is good enough, I believe).